ctapipe/configuration/configuration.py

```diff
@@ -0,0 +1,312 @@
+'''
```

Comment:
Having `configuration/configuration.py` can lead to confusion.
What Astropy does is instead call it `configuration/core.py` ... OK to rename?